src/weekly/query.js

```diff
@@ -103,7 +105,7 @@ async function queryZuora (deliveryDate, config: Config) {
       WHERE 
        (Subscription.Status = 'Active' OR Subscription.Status = 'Cancelled') AND
          Product.ProductType__c = 'Guardian Weekly' AND
-        ( RatePlan.Name = 'Guardian Weekly 6 Issues' OR RatePlan.Name = 'Guardian Weekly 12 Issues' ) AND
+        ( RatePlan.Name = 'Guardian Weekly 6 Issues' OR RatePlan.Name = 'Guardian Weekly 12 Issues' OR RatePlan.Name = 'GW Oct 18 - Six for Six - ROW' OR RatePlan.Name = 'GW Oct 18 - Six for Six - Domestic' ) AND
```

Comment:
we should really add a custom field to use as an id instead of relying on the name of the rateplan